containers/ecr-viewer/src/app/admin/user/UserForm.tsx

```diff
@@ -0,0 +1,288 @@
+
+  return (
+    <FormPageContent
+      action={`${action} user`}
+      formValid={valid}
+      submitAction={async () => {
+        const res = await submitAction(
+          email.trim(),
+          userType,
+          selectedPrograms,
+        );
+        if (!res.error)
+          createToast(`${email.trim()} successfully saved`, "success");
+        return res;
+      }}
+      successRoute="/admin/user"
+    >
+      <EmailFieldSet email={email} setEmail={setEmail} />
+      <UserTypeFieldSet userType={userType} setUserType={setUserType} />
+      <ProgramFieldSet
+        programs={programs}
+        setPrograms={setPrograms}
+        numProgramsSelected={numProgramsSelected}
+        userType={userType}
+      />
+    </FormPageContent>
+  );
+};
+
+const EmailFieldSet = ({
+  email,
+  setEmail,
+}: {
+  email: string;
+  setEmail: (n: string) => void;
+}) => {
+  return (
+    <FieldSet legend="Email">
+      <span>
+        Add the new user by their login email (<RequiredMarker />)
+      </span>
+      <label className="usa-label">
+        EMAIL
+        <RequiredMarker />
+        <TextInput
+          type="email"
+          required={true}
+          id="email"
+          name="email"
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}
+        />
+      </label>
+    </FieldSet>
+  );
+};
+
+const UserTypeFieldSet = ({
+  userType,
+  setUserType,
+}: {
+  userType: UserType;
+  setUserType: (n: UserType) => void;
+}) => {
+  return (
+    <FieldSet legend="User type">
+      <label className="display-block width-full">
+        USER TYPE
+        <RequiredMarker />
+        {[
+          {
+            name: "admin",
+            description:
+              "Admins have full access to user management, program management, and the eCR Library",
+          },
+          {
+            name: "standard",
+            description:
+              "Standard users can only use the eCR Library with limited access to program area(s)",
+          },
+        ].map((option) => (
+          <Radio
+            key={option.name}
+            label={toTitleCase(option.name)}
+            labelDescription={option.description}
+            type="radio"
+            required={true}
+            name="userType"
+            id={`userType-${option.name}`}
+            value={option.name}
+            checked={userType === option.name}
+            onChange={(e) => setUserType(e.target.value as UserType)}
+          />
+        ))}
+      </label>
+    </FieldSet>
+  );
+};
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export const ProgramFieldSet = ({
+  programs,
+  setPrograms,
+  numProgramsSelected,
+  userType,
+}: {
+  programs: FormProgram[];
+  setPrograms: (c: FormProgram[]) => void;
+  numProgramsSelected: number;
+  userType: UserType;
+}) => {
+  const [expandedPrograms, setExpandedPrograms] = useState<
+    Record<string, boolean>
+  >(valsToBoolean(programs, true));
```

Comment:
should these ones start collapsed instead of expanded? the designs I think indicate that they should
```suggestion
  const [expandedPrograms, setExpandedPrograms] = useState<
    Record<string, boolean>
  >(valsToBoolean(programs, false));
```

Reply:
you're so right!